core/src/main/java/nl/nn/adapterframework/lifecycle/IbisApplicationContext.java

```diff
@@ -157,6 +173,42 @@ private XmlWebApplicationContext createWebApplicationContext() throws BeansExcep
 		return applicationContext;
 	}
 
+	/**
+	 * Loads springContext, springUnmanagedDeployment, springCommon and files specified by the SPRING.CONFIG.LOCATIONS
+	 * property in AppConstants.properties
+	 * 
+	 * @param classLoader to use in order to find and validate the Spring Configuration files
+	 * @return A String array containing all files to use.
+	 */
+	private String[] getSpringConfigurationFiles(ClassLoader classLoader) {
+		List<String> springConfigurationFiles = new ArrayList<String>();
+		springConfigurationFiles.add(XmlWebApplicationContext.CLASSPATH_URL_PREFIX + "/springContext.xml");
+		springConfigurationFiles.add(XmlWebApplicationContext.CLASSPATH_URL_PREFIX + "/springUnmanagedDeployment.xml");
+		springConfigurationFiles.add(XmlWebApplicationContext.CLASSPATH_URL_PREFIX + "/springCommon.xml");
+
+		StringTokenizer locationTokenizer = AppConstants.getInstance().getTokenizedProperty("SPRING.CONFIG.LOCATIONS");
+		while(locationTokenizer.hasMoreTokens()) {
+			String file = locationTokenizer.nextToken();
+			log.info("found spring configuration file to load ["+file+"]");
+
+			URL fileURL = classLoader.getResource(file);
+			if(fileURL == null) {
+				log.error("unable to locate Spring configuration file ["+file+"]");
+				System.out.println("unable to locate Spring configuration file ["+file+"]");
+			} else {
+				if(file.indexOf(":") == -1) {
+					file = XmlWebApplicationContext.CLASSPATH_URL_PREFIX+"/"+file;
+				}
+
+				springConfigurationFiles.add(file);
+			}
+		}
+
+		log.info("loading Spring configuration files "+springConfigurationFiles+"");
+		System.out.println("loading Spring configuration files "+springConfigurationFiles+"");
```

Comment:
Deze System.out.println is niet echt nodig, toch? Anders in een comment zetten waarom hier wel.